test/index.spec.ts

```diff
@@ -0,0 +1,23 @@
+import * as dotenv from 'dotenv';
+import axios from 'axios';
+import { expect } from 'chai';
+import { startServer } from '../src/start-server';
+
+dotenv.config({ path: './test.env' });
+
+before('Starting server', async () => {
+  await startServer();
+});
```

Comment:
Por exemplo aqui. Se o `startServer` não for finalizado apenas quando a conexão com o banco for feita, ele pode começar a executar os testes antes de ter conectado com o banco, certo?